src/core/client/admin/components/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { StatelessComponent } from "react";
+import { HorizontalGutter } from "talk-ui/components";
+import styles from "./App.css";
+import Navigation from "./Navigation";
+
+const App: StatelessComponent = ({ children }) => (
+  <div className={styles.root}>
+    <HorizontalGutter>
```

Comment:
`HorizontalGutter` can be used as root and we would save one unnecessary `div`, e.g. `<HorizontalGutter className={styles.root}>`